lib/ui/widgets/authenticator_list_tile.dart

```diff
@@ -32,27 +35,19 @@ class AuthenticatorListTile extends StatelessWidget {
       });
     }
 
-    final descriptions = {
-      Aaid.pin.rawValue: localization.authenticatorDescriptionPin,
-    };
-
-    if (Platform.isAndroid) {
-      descriptions.addAll({
-        Aaid.fingerprint.rawValue:
-            localization.authenticatorDescriptionFingerprint,
-        Aaid.biometric.rawValue: localization.authenticatorDescriptionBiometric,
-      });
-    } else if (Platform.isIOS) {
-      descriptions.addAll({
-        Aaid.fingerprint.rawValue: localization.authenticatorDescriptionTouchID,
-        Aaid.biometric.rawValue: localization.authenticatorDescriptionFaceID,
-      });
+    String subTitle = "";
+    if (!item.isEnabled()) {
+      if (!item.isPolicyCompliant) {
+        subTitle =
+            localization.selectAuthenticatorAuthenticatorIsNotPolicyCompliant;
+      } else if (!item.isUserEnrolled) {
+        subTitle = localization.selectAuthenticatorAuthenticatorIsNotEnrolled;
+      }
     }
 
     return ListTile(
-      title: Text(
-          titles[authenticator.aaid] ?? 'Unknown AAID: ${authenticator.aaid}'),
-      subtitle: Text(descriptions[authenticator.aaid] ?? 'N/A'),
+      title: Text(titles[item.aaid] ?? 'Unknown AAID: ${item.aaid}'),
```

Comment:
Should I change the Android implementation this way? I throw an IllegalArgumentException at this moment.

Reply:
I don't think it is really important...